tests/conftest.py

```diff
@@ -13,7 +13,7 @@ def test_client():
         db.create_all()
         db.session.commit()
 
-        from build_database import build_database
+        from orm.entities.Election.election_helper import build_database
```

Comment:
Due to the refactoring, the testing strategy also has be changed. I'm creating another ticket to fix that.
https://github.com/ECLK/results-tabulation-api/issues/173
CC: @callkalpa